src/main/steps/common/progress-bar.njk

```diff
@@ -9,14 +9,14 @@
     label: {
       html: textWithVisuallyHiddenSpan(submitted, displayState.isAtOrAfter('Submitted'))
     },
-    complete: displayState.isAtOrAfter('Submitted'),
-    active: displayState.at('AwaitingDocuments')
+    complete: displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'),
+    active: displayState.at('AwaitingDocuments') and not (displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'))
```

Comment:
Just checking - do we need the second condition after "and not" to determine whether it is "active", as we check for a specific match against AwaitingDocuments for the first part of the condition?

Reply:
Discussed this on Slack and we will leave it as is